os/arch/arm/src/stm32/stm32_userspace.c

```diff
@@ -74,6 +74,14 @@
  * Private Data
  ****************************************************************************/
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+#ifdef CONFIG_SUPPORT_COMMON_BINARY
```

Comment:
Except calling mpu init from this file, others are the same as up_userspace.c. How about moving mpu init function at somewhere and removing this?

Reply:
Managing several almost the same file is not good. That's why I suggest.